### PR TITLE
Update royal-tsx to 3.0.0.1000

### PR DESCRIPTION
**Pull request by artdevjs**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/royal-tsx.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 cask 'royal-tsx' do
-  version '2.2.9.1000'
-  sha256 '7fccf594f56fd2824efef0bc933669ad3ab2f3788224450a65fd2eb493423bf9'
+  version '3.0.0.1000'
+  sha256 'cd2fce3697f9b414285b2631587d4c2449e0f45a686f1869288e2ac11de958ff'
 
-  url "https://royaltsx-v2.royalapplications.com/updates/royaltsx_#{version}.dmg"
+  url "https://royaltsx-v3.royalapplications.com/updates/royaltsx_#{version}.dmg"
```
Comment (vitorgalvao) [inline Casks/royal-tsx.rb]: `url "https://royaltsx-v#{version.major}.royalapplications.com/updates/royaltsx_#{version}.dmg"`

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by miccal** [comment]
`brew cask style` is failing as the comment above the `url` stanza does not match due to the interpolation, but otherwise everything else is fine.

**Review by miccal — APPROVED** [review]

**Comment by vitorgalvao** [comment]
@miccal You’re right, hadn’t noticed. Apologies, @artdevjs, mind changing it back? Since the major number changes only sporadically, it’s preferable to have the cask not break on checks.

I remember we had similar cases when introducing this cop. Did we have a better solution, @reitermarkus?

**Comment by vitorgalvao** [comment]
Closed by mistake.

**Comment by adidalal** [comment]
https://github.com/caskroom/homebrew-cask/pull/28069

**Pull request closed without merging**